### PR TITLE
Add expression example

### PR DESCRIPTION
**Pull request by DelazJ**

### CODE REVIEW

File: docs/user_manual/working_with_vector/expression.rst [hunk 1]
```diff
@@ -56,6 +56,10 @@ The Expression builder dialog offers access to the:
          WHEN "pop_density" >= 150 THEN 'High population density'
     END
 
+* Update a point layer field with values of overlapping features in a polygon layer::
+
+    aggregate('polygonlayername', 'concatenate', "fieldtoextract", contains($geometry, geometry(@parent)))
```
Comment (havatv) [inline docs/user_manual/working_with_vector/expression.rst:61]: I can't make this work.
In which context does it work?

File: docs/user_manual/working_with_vector/expression.rst [hunk 2]
```diff
@@ -56,6 +56,10 @@ The Expression builder dialog offers access to the:
          WHEN "pop_density" >= 150 THEN 'High population density'
     END
 
+* Update a region layer field with comma separated name of the overlapping airports::
```
Comment (havatv) [inline docs/user_manual/working_with_vector/expression.rst:59]:
```suggestion
* Update a region layer field with the names (comma separated) of contained airports::
```

**Review by havatv — COMMENTED** [review]
I suggest a reformulation.

File: docs/user_manual/working_with_vector/expression.rst (hunk 2 above)
Comment (DelazJ) [reply docs/user_manual/working_with_vector/expression.rst:59]: if you want

**Pull request merged**